### PR TITLE
[Docs] update Japanese translation of other_eclipse.md

### PR DESCRIPTION
**Pull request by umi-umi**
## Description

update Japanese translation of docs/other_eclipse.md.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@shelaf @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

File: docs/ja/other_eclipse.md [hunk 1]
```diff
@@ -78,16 +78,12 @@ Workspace 選択で入力を促された場合は、Eclipse メタデータと
...
-2. フォルダツリーを選択したキーボードまで展開します。例えば、`qmk_firmware/keyboards/ergodox`
-3. キーボードフォルダを右クリックして、<kbd>New…</kbd> を選択します (あるいはフォルダを選択し、ツリーの上にある <kbd>New Make Target</kbd> アイコンをクリックします)
-4. ビルド target の名前を選択します。例えば、_clean \<your keymap\>_
-5. Make Target: これはコマンドラインからビルドする時に `make` に渡す引数です。target 名がこれらの引数と一致しない場合は、<kbd>Same as target name</kbd> のチェックを外し、正しい引数を入力します。例えば、`clean <your keymap>`
-6. 他のオプションはチェックしたままにして、<kbd>OK</kbd> をクリックします。これで、選択されたキーボードの下に、make target が表示されます。
-7. (オプション) target ツリーの上にある <kbd>Hide Empty Folders</kbd> アイコンボタンを、ビルド target だけが表示されるように切り替えます。
-8. 作成したビルド target をダブルクリックし、ビルドを起動します。
-9. 下部の <kbd>Console</kbd> ビューを選択し、実行中のビルドを眺めます。
+
+プロジェクトのデフォルトの make 対象を `all` から私たちが取り組んでいる特定のキーボードとキーマップ、例えば  `kinesis/kint36:stapelberg` に変更します。このように、プロジェクトのクリーニングやビルドのようなプロジェクト全体のアクションは、長い時間を掛けたり、Eclipse を完全にロックしたりすることなく、迅速に完了します。
```
Comment (shelaf) [inline docs/ja/other_eclipse.md:82]:
```suggestion
プロジェクトのデフォルトの make 対象を `all` から私たちが取り組んでいる特定のキーボードとキーマップの組み合わせ、例えば  `kinesis/kint36:stapelberg` に変更します。このように、プロジェクトのクリーニングやビルドのようなプロジェクト全体のアクションは、長い時間を掛けたり、Eclipse を完全にロックしたりすることなく、迅速に完了します。
```

**Review by shelaf — APPROVED** [review]

File: docs/ja/other_eclipse.md [hunk 2]
```diff
@@ -78,16 +78,12 @@ Workspace 選択で入力を促された場合は、Eclipse メタデータと
...
-2. フォルダツリーを選択したキーボードまで展開します。例えば、`qmk_firmware/keyboards/ergodox`
-3. キーボードフォルダを右クリックして、<kbd>New…</kbd> を選択します (あるいはフォルダを選択し、ツリーの上にある <kbd>New Make Target</kbd> アイコンをクリックします)
-4. ビルド target の名前を選択します。例えば、_clean \<your keymap\>_
-5. Make Target: これはコマンドラインからビルドする時に `make` に渡す引数です。target 名がこれらの引数と一致しない場合は、<kbd>Same as target name</kbd> のチェックを外し、正しい引数を入力します。例えば、`clean <your keymap>`
-6. 他のオプションはチェックしたままにして、<kbd>OK</kbd> をクリックします。これで、選択されたキーボードの下に、make target が表示されます。
-7. (オプション) target ツリーの上にある <kbd>Hide Empty Folders</kbd> アイコンボタンを、ビルド target だけが表示されるように切り替えます。
-8. 作成したビルド target をダブルクリックし、ビルドを起動します。
-9. 下部の <kbd>Console</kbd> ビューを選択し、実行中のビルドを眺めます。
+
+プロジェクトのデフォルトの make 対象を `all` から私たちが取り組んでいる特定のキーボードとキーマップの組み合わせ、例えば `kinesis/kint36:stapelberg` に変更します。このように、プロジェクトのクリーニングやビルドのようなプロジェクト全体のアクションは、長い時間を掛けたり、Eclipse を完全にロックしたりすることなく、迅速に完了します。
```
Comment (mtei) [inline docs/ja/other_eclipse.md:82]: `This way,` をこう訳していいのか自信がいまひとつありませんが、こう解釈するとしっくりする気がします。

```suggestion
プロジェクトのデフォルトの make 対象を `all` から私たちが取り組んでいる特定のキーボードとキーマップの組み合わせ、例えば `kinesis/kint36:stapelberg` に変更します。このようにすると、プロジェクトのクリーニングやビルドのようなプロジェクト全体のアクションは迅速に完了し、長い時間がかかったり Eclipse が完全にロックしたりすることがなくなります。
```

**Review by mtei — APPROVED** [review]

**Comment by mtei** [comment]
I think this is ready to merge.

**Comment by mtei** [comment]
Thanks!

**Pull request merged**